dockerfiles/Dockerfile

```diff
@@ -55,6 +55,9 @@ RUN echo "deb http://deb.debian.org/debian bookworm-backports main" >> /etc/apt/
     flex \
     bison \
     libyaml-dev \
+    libboost-system-dev \
+    libboost-thread-dev \
+    libboost-filesystem-dev \
```

Comment:
We're including these development libraries but not actually enabling the plugin itself - is that right?

Reply:
Ah, we globally enabled it which may be an entirely separate issue.

Reply:
So, I removed the global enabling of the plugin. 

I kept these boost libraries here (they are required to build the vsomeip code) to allow enabling the SOME/IP input plugin (i.e. `-DFLB_IN_SOMEIP=On`)